tests/pyccel/scripts/classes/classes_2_C.py

```diff
@@ -2,13 +2,14 @@
 import numpy as np
 
 #$ header class Point(public)
-#$ header method __init__(Point)
+#$ header method __init__(Point, float, float)
```

Comment:
Why not create a new test rather than modifying this one to try and cover everything at once? Having multiple incremental tests will make it easier to locate a bug if something breaks

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1472/commits/2f3a91f4cec9be0463fa0f1621cc76fa8c46b5b2